src/main/kotlin/org/pkl/intellij/util/Util.kt

```diff
@@ -45,8 +45,8 @@ val pklDir: VirtualFile?
 val pklCacheDir: VirtualFile?
   get() = pklDir?.findChild("cache")
 
-val packages1CacheDir: VirtualFile?
-  get() = pklCacheDir?.findChild("package-1")
+val packagesCacheDir: VirtualFile?
+  get() = pklCacheDir?.run { findChild("package-2") ?: findChild("package-1") }
```

Comment:
This commits to the newest cache directory found on the system. For backward compatibility, shouldn't this also depend on the `MinPklVersion` of a module (for example)?

Reply:
I think the ability to discover sources is different than providing diagnostics about code correctness. It makes sense to me that searching for sources is a "best-effort" approach, and I don't think this needs to change based on `minPklVersion`.